hoot-core/src/main/cpp/hoot/core/io/OsmGeoJsonWriter.h

```diff
@@ -44,7 +43,7 @@ namespace hoot
  *
  *
  */
-class OsmGeoJsonWriter : public OsmJsonWriter, public Configurable
+class OsmGeoJsonWriter : public OsmJsonWriter
```

Comment:
`OsmJsonWriter` (the base class) is now derived from `Configurable` so it can't be inherited again.